docs/UserGuide.md

```diff
@@ -141,7 +141,7 @@ download ClientCare.
 
 6. Refer to the [Features](#features) below for details of each command.
 
-<br>
```

Comment:
Would it be better to separate out user guide changes from functional code changes in different pull requests in the future?

Reply:
Okay, I'll take note!